roles/ceph-osd/tasks/main.yml

```diff
@@ -28,6 +28,7 @@
     - item.value.removable == "0"
     - item.value.partitions|count == 0
     - item.value.holders|count == 0
+    - "'dm-' not in item.key"
```

Comment:
this syntax is not correct
`'dm-' not in item.key` implies `item.key` is an array so we would want to check one of the element of that array is not a string equal to `'dm-'`